include/snowflake/client.h

```diff
@@ -277,7 +277,10 @@ typedef enum SF_GLOBAL_ATTRIBUTE {
     SF_GLOBAL_CA_BUNDLE_FILE,
     SF_GLOBAL_SSL_VERSION,
     SF_GLOBAL_DEBUG,
-    SF_GLOBAL_OCSP_CHECK
+    SF_GLOBAL_OCSP_CHECK,
+    SF_GLOBAL_CLIENT_CONFIG_FILE,
+    SF_GLOBAL_LOG_LEVEL,
```

Comment:
remind me why we need SF_GLOBAL_LOG_LEVEL and SF_GLOBAL_LOG_PATH?

Reply:
Updated description